client/components/accordion/index.jsx

```diff
@@ -1,9 +1,9 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	noop = require( 'lodash/noop' ),
-	classNames = require( 'classnames' );
+import React from 'react';
+import noop from 'lodash/noop';
```

Comment:
FYI: We also tend to prefer importing from the root Lodash module, which was only possible more recently with the merging of #6539:

```js
import { noop } from 'lodash';
```

Not a blocker, but figured it was worth mentioning as a general refactoring point.